buildconfigs/oak_restricted_kernel_simple_io_init_rd_wrapper.toml

```diff
@@ -6,4 +6,4 @@ command = [
   "just",
   "oak_restricted_kernel_simple_io_init_rd_wrapper",
 ]
-artifact_path = "./oak_restricted_kernel_wrapper/target/x86_64-unknown-none/release/oak_restricted_kernel_simple_io_init_rd_wrapper_bin"
+artifact_path = "./oak_restricted_kernel_wrapper/target/released_bin_with_components_oak_restricted_kernel_simple_io_init_rd/*"
```

Comment:
in theory this means the slsa builder should record all three binaries in this folder as the provenance subject: `bzImage, kernel_setup_data, kernel_image`

https://github.com/slsa-framework/slsa-github-generator/blob/e8c2dcff94b830dfe6897c48b7218c85fe6f3eb3/internal/builders/docker/README.md?plain=1#L150